.prettierignore

```diff
@@ -0,0 +1,11 @@
+LICENSE.md
```

Comment:
I am excluding this file to avoid whitespace-only changes in the license text and to keep the file consistent with what we have in other repositories.